gm4_rituals/data/rituals/functions/init.mcfunction

```diff
@@ -0,0 +1,29 @@
+#announce module installation
+tellraw @a[gamemode=creative] [{"translate":"%1$s","with":["[GM4]: ",{"translate":"text.gm4.prefix"}]},{"translate":"%1$s","with":["Installing Rituals...",{"translate":"text.gm4.modules.update.installing","with":["Rituals"]}]}]
+execute unless entity @a[limit=1] run say GM4: Installing Rituals...
+
+#declare and initialise scoreboards and settings
+scoreboard players set update_happened gm4_up_check 1
+scoreboard players set rituals gm4_modules 1
+scoreboard players set rituals gm4_clock_tick 0
+scoreboard players set rituals_4 gm4_clock_tick 4
+scoreboard players set rituals_8 gm4_clock_tick 8
+scoreboard players set rituals_12 gm4_clock_tick 12
+
+scoreboard objectives add gm4_rit_sneak minecraft.custom:minecraft.sneak_time
+scoreboard objectives add gm4_rit_stock dummy
+scoreboard objectives add gm4_rit_tick dummy
+
+scoreboard objectives add gm4_rit_r_clwe dummy
+scoreboard objectives add gm4_rit_r_rain dummy
+scoreboard objectives add gm4_rit_r_thun dummy
```

Comment:
These objective names are not very descriptive (expanded it would be `gm4_rituals_ritual_clearweather`). So just use `gm4_rit_clear`, `gm4_rit_rain` and `gm4_rit_thunder`